tests/dags/test_invalid_param_int.py

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.models.param import Param
+from airflow.operators.python import PythonOperator
+
+with DAG(
+    "test_invalid_param_int",
+    start_date=datetime(2021, 1, 1),
+    schedule_interval="@once",
+    params={
+        # integer param with above limit default value
+        "int_param": Param(10, type="integer", minimum=2, maximum=4),
+    },
```

Comment:
Take this for example: The scheduler is going to try and create a run as soon as it's unpaused, but it will never be able to as this param has no value.